dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -314,12 +314,12 @@ if [ "$RH_CHE_USE_CUSTOM_REGISTRIES" == "true" ]; then
     oc delete configmap plugin-registry > /dev/null 2>&1
   fi
   echo "Downloading che plugin registry yaml"
-  if ! (curl -L0fs https://raw.githubusercontent.com/eclipse/che-plugin-registry/master/deploy/openshift/che-plugin-registry.yml -o che-plugin-registry.yaml > /dev/null 2>&1); then
+  if ! (curl -L0fs https://raw.githubusercontent.com/eclipse/che-plugin-registry/7.26.0/deploy/openshift/che-plugin-registry.yml -o che-plugin-registry.yaml > /dev/null 2>&1); then
     echo -e "\\033[91;1mCould not download che-plugin-registry yaml!\\033[0m"
     exit 2
   fi
   echo "Downloading che devfile registry yaml"
-  if ! (curl -L0fs https://raw.githubusercontent.com/eclipse/che-devfile-registry/master/deploy/openshift/che-devfile-registry.yaml -o che-devfile-registry.yaml > /dev/null 2>&1); then
+  if ! (curl -L0fs https://raw.githubusercontent.com/eclipse/che-devfile-registry/7.26.0/deploy/openshift/che-devfile-registry.yaml -o che-devfile-registry.yaml > /dev/null 2>&1); then
```

Comment:
replace `7.26.0` with `${RH_CHE_VERSION}`

Reply:
this is not fix we do not expect any more rollouts
and in fact this replacemnt should have been done from the very begging, I do not understand why have you opted for the fancy `yq` logic with replacing images

Reply:
Hard to say, it was implemented 16 months ago...